scilpy/tractanalysis/grid_intersections.pyx

```diff
@@ -30,6 +30,22 @@ cdef struct Pointers:
 @cython.wraparound(False)
 @cython.cdivision(True)
 def grid_intersections(streamlines):
+    """
+    Cut streamlines segments into smaller segments such that a segment covering
+    more than one voxel is split into smaller segments that either end or start
+    at voxel boundaries.
```

Comment:
Good description. I would rename the function something like "regrid_streamlines_at_voxel_faces" or something.